qiskit/transpiler/passes/width.py

```diff
@@ -22,4 +22,4 @@ class Width(AnalysisPass):
     """
 
     def run(self, dag):
-        self.property_set['width'] = dag.width()
+        self.property_set['width'] = dag.num_qubits()
```

Comment:
this pass name should probably be changed too, to be consistent.

Reply:
@ajavadia ... The class name, file name and all?

Reply:
yes. because this pass no longer reports width. it reports num_qubits.

I would just keep the `width` pass and add a new pass for `num_qubits`

Reply:
New pass added. `width` pass restored to previous.